src/store-tree/index.ls

```diff
@@ -40,45 +37,34 @@ class StoreTree extends StoreNode
 
 
   $is-loading: ->
-    for own key of @_children when @[key].$is-loading!
-      return yes
-    no
+    @_children |> values |> any (.$is-loading!)
```

Comment:
My point is, in JavaScript land "falsy" is a proper value. You can document that the library returns falsy, and that's perfectly legit and idiomatic. JS takes a light approach towards type systems, this type of flexibility one central aspect of it.